### PR TITLE
From orbit, nuke the Telnet connection and all supporting infra.

### PR DESCRIPTION
**Pull request by DHowett**
This is not going to be our plan of record for Universal going forward.

This updates the Universal configuration to 1) match non-universal and 2) switch to local applications

### CODE REVIEW

**Comment by DHowett** [comment]
I also got rid of the "we have two versions of GSL" nonsense :)

File: src/cascadia/TerminalSettingsModel/defaults-universal.json [hunk 1]
```diff
@@ -78,14 +106,21 @@
         { "command": "closeWindow", "keys": "alt+f4" },
         { "command": "toggleFullscreen", "keys": "alt+enter" },
         { "command": "toggleFullscreen", "keys": "f11" },
+        { "command": "toggleFocusMode" },
+        { "command": "toggleAlwaysOnTop" },
```
Comment (zadjii-msft) [inline src/cascadia/TerminalSettingsModel/defaults-universal.json:110]: Oh man will these work in universal?

**Review by zadjii-msft — APPROVED** [review]
I still can't believe that this works

**Comment by zadjii-msft** [comment]
why is our CI falling to pieces as of late?

**Comment by DHowett** [comment]
Excellent question! The memory issue is because I re-enabled parallel builds (sorry :|). This new weird issue? _no idea_.

File: src/cascadia/TerminalSettingsModel/defaults-universal.json (hunk 1 above)
Comment (DHowett) [reply src/cascadia/TerminalSettingsModel/defaults-universal.json:110]: nope, but there's also no point in leaving them off. Many things act _weird_ in WTU, but we should undiverge where we can :)

**Comment by DHowett** [comment]
The x64 build failed for an unrelated reason: 

image

**Comment by codeofdusk** [comment]
I like the branch name!

**Comment by DHowett** [comment]
Thanks! :smile:

**Comment by miniksa** [comment]
https://unreal.fandom.com/wiki/Ion_Painter

**Review by miniksa — APPROVED** [review]

**Pull request merged**